icechunk-python/python/icechunk/__init__.py

```diff
@@ -444,10 +420,6 @@ def async_ancestry(self) -> AsyncGenerator[SnapshotMetadata, None]:
         """
         return self._store.async_ancestry()
 
-    async def empty(self) -> bool:
-        """Check if the store is empty."""
-        return await self._store.empty()
```

Comment:
what happened here? did this disappear from the abc?

Reply:
Its not the same, takes prefix now. Adding to backlog

Reply:
I added a simple implmenetation that just uses the rust `list_dir` function. I think that is right, but there is probably a more optimized way to do it?

Reply:
This is great, it will get fast once we make list_dir fast. Thank you